### ISSUE TITLE
Windows cant dock to side when toggle titlebar

### ISSUE THREAD

**Issue by brunotvs** [ahrm/sioyek#1125]
When I use toggle titlebar command, I cant dock to the side in windows (winkey+left|rigth arrow)

**Comment by ahrm** [comment]
I think this is an issue with Qt (it doesn't support titlebar-less windows very well). I don't think there is much I can do about it, hopefully it will be fixed in future Qt versions.

**Comment by brunotvs** [comment]
Is it possible to change the title bar colocar then?

**Comment by ahrm** [comment]
Currently there is no option for that on windows.

**Comment by CKPEIIKA** [comment]
For me its not working either with titlebar or not... So i need to resize windows manually each time.

**Comment by ahrm** [comment]
Yes, qt windows without titlebar seem to not be supported very well on windows. There is not much I can do about that, this is a qt bug.
